package.json

```diff
@@ -27,8 +27,9 @@
     "@bonfida/name-offers": "0.0.10",
     "@bonfida/name-tokenizer": "0.0.12",
     "@bonfida/sns-emitter": "0.1.7",
-    "@bonfida/sns-react": "2.0.2",
-    "@bonfida/spl-name-service": "2.0.3",
+    "@bonfida/sns-react": "2.0.3",
+    "@bonfida/sns-records": "^0.0.1-alpha.8",
```

Comment:
We need to avoid using `^` in versions, because from my experience it might lead to build problems at random time and it will be very time-consuming to understand what happened. Also, in React Native world from what I googled and asked people all the dependencies should be fixed, because it's a very fragile environment